glue-codes/openfast-cpp/src/OpenFAST.cpp

```diff
@@ -820,8 +830,10 @@ void fast::OpenFAST::allocateMemory() {
     cDriver_Input_from_FAST.resize(nTurbinesProc) ;
     cDriver_Output_to_FAST.resize(nTurbinesProc) ;
     
-  if(!scStatus)
-      scio.from_SC.resize(nTurbinesProc);
+    if(scStatus) {
```

Comment:
@gantech was the `!scStatus` a bug or is it meant to resize this vector when the SC is not used?